Data/DataModel/PlayerInEvent.cs

```diff
@@ -20,8 +20,10 @@ public class PlayerInEvent
 
         public string Lunch { get; set; }
 
+        [MaxLength(100)]
         public string LunchModifications { get; set; }
 
+        [MaxLength(5)]
```

Comment:
This might be a word, maybe a 20 character limit?

Reply:
So changed